app/helpers/network_port_helper/textual_summary.rb

```diff
@@ -45,8 +53,9 @@ def textual_ems_network
   def textual_instance
     label    = ui_lookup(:table => "vm_cloud")
     instance = @record.device
-    h        = {:label => label, :image => "vm"}
+    h        = nil
```

Comment:
but plan is to get rid of this entirely, instance and template have messed up autodiscovery of rbac, otherwise I would just write here:

def textual_instance
   @record.device
end